client/my-sites/plan-features/header.jsx

```diff
@@ -3,23 +3,24 @@
  */
 import React, { Component, PropTypes } from 'react';
 import noop from 'lodash/noop';
+import identity from 'lodash/identity';
 
 /**
  * Internal Dependencies
  **/
 import { localize } from 'i18n-calypso';
 import Gridicon from 'components/gridicon';
+import PlanFeaturesPrice from './price';
 import {
 	PLAN_FREE,
 	PLAN_PREMIUM,
 	PLAN_BUSINESS
 } from 'lib/plans/constants';
 
-class PlanFeaturesHeader extends Component {
+export class PlanFeaturesHeader extends Component {
 
 	render() {
-		const { billingTimeFrame, current, planType, popular, price, title, translate } = this.props;
-		const isFree = planType === PLAN_FREE;
+		const { billingTimeFrame, currencyCode, current, planType, popular, rawPrice, title, translate } = this.props;
```

Comment:
Might be a good opportunity to split into one line per variable, maybe not.

```js
const {
	billingTimeFrame,
	currencyCode,
	current,
	planType,
	…
} = this.props;
```

pleased to see it's alphabetical already though!